packages/ui/src/components/topbar.tsx

```diff
@@ -18,7 +18,7 @@ const Topbar = {
     return (
       <div
         className={cx(
-          `grid w-full ${gridCols} font-regular h-[var(--cn-breadcrumbs-height)] topbar-bottom-border items-center gap-6 px-5 text-sm`,
+          `grid w-full ${gridCols} font-regular h-[var(--cn-breadcrumbs-height)] items-center gap-6 px-5 text-sm`,
```

Comment:
I think we can kill off this template string
```suggestion
          'grid w-full font-regular h-[var(--cn-breadcrumbs-height)] items-center gap-6 px-5 text-sm',
          hasCenter ? 'grid-cols-[auto_1fr_auto]' : 'grid-cols-[1fr_auto]',
```

Reply:
FIxed